### PR TITLE
Use kernel and u- boot from BSP

### PR DESCRIPTION
**Pull request by spanceac**

### CODE REVIEW

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend [hunk 1]
```diff
@@ -2,21 +2,18 @@ inherit kernel-resin
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+LINUX_VERSION = "4.19.44"
+
 SRC_URI_append = " \
-	file://0001-Revert-cgroup-Disable-cgroup-memory-by-default.patch \
```
Comment (agherzan) [inline layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend]: I see we removed this patch from SRC_URI and haven't removed it as a file? Also, shouldn't this still be needed? I don't expect they reverted it upstream.

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend [hunk 2]
```diff
@@ -2,21 +2,18 @@ inherit kernel-resin
...
 	file://0002-wireless-wext-Bring-back-ndo_do_ioctl-fallback.patch \
 	file://0003-leds-pca963x-Fix-MODE2-initialization.patch \
 	file://0001-Add-npe-x500-m3-overlay.patch \
 	file://0002-NFLX-2019-001-SACK-Panic.patch \
-	file://0003-NFLX-2019-001-SACK-Panic-for-lteq-4.14.patch \
```
Comment (agherzan) [inline layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend]: Could you please add the reason (in git log) why removed this patch?

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend [hunk 3]
```diff
@@ -2,21 +2,18 @@ inherit kernel-resin
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+LINUX_VERSION = "4.19.44"
```
Comment (agherzan) [inline layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend]: Why do we set another version here?

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend (hunk 1 above)
Comment (spanceac) [reply layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend]: memory cgroups are activated by default in the new kernel version:
root@f026b3a:~# cat /proc/cgroups 
#subsys_name    hierarchy       num_cgroups     enabled
cpuset  8       4       1
cpu     6       4       1
cpuacct 6       4       1
blkio   5       4       1
memory  9       98      1
devices 2       10      1
freezer 4       4       1
net_cls 7       4       1
pids    3       40      1

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend (hunk 2 above)
Comment (spanceac) [reply layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend]: All Netflix patches are applied upstream now, so Andrei's comment is obsolete

File: layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend (hunk 3 above)
Comment (spanceac) [reply layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi_%.bbappend]: Fixed

**Comment by floion** [comment]
@spanceac can you rebase this PR if it is still applicable?

**Comment by spanceac** [comment]
Merged in a different PR

**Pull request closed without merging**